applications/asset_tracker_v2/prj.conf

```diff
@@ -49,83 +49,12 @@ CONFIG_LTE_PSM_REQ_RAT="00001010"
 # Modem info library to obtain information about network and device
 CONFIG_MODEM_INFO=y
 
-# GNSS
-## Uncomment to use external antenna for GNSS.
-#CONFIG_GPS_MODULE_ANTENNA_EXTERNAL=y
-
-# Cloud backend - To use a specific cloud backend uncomment the corresponding
-#                 section.
-
-## AWS IoT
-## Mandatory application specific configurations for AWS_IOT.
-# CONFIG_AWS_IOT=y
-# CONFIG_AWS_IOT_TOPIC_UPDATE_DELTA_SUBSCRIBE=y
-# CONFIG_AWS_IOT_TOPIC_GET_REJECTED_SUBSCRIBE=y
-# CONFIG_AWS_IOT_AUTO_DEVICE_SHADOW_REQUEST=y
-# CONFIG_AWS_IOT_MQTT_RX_TX_BUFFER_LEN=2048
-# CONFIG_AWS_IOT_MQTT_PAYLOAD_BUFFER_LEN=2048
-# CONFIG_AWS_IOT_APP_SUBSCRIPTION_LIST_COUNT=3
-# CONFIG_AWS_IOT_CLIENT_ID_APP=y
-# CONFIG_AWS_FOTA=y
-
-## AWS IoT options that must be configured.
-# CONFIG_AWS_IOT_SEC_TAG=42
-# CONFIG_AWS_IOT_BROKER_HOST_NAME=""
-
-# # Azure IoT Hub
-# ## Mandatory application specific configurations for AZURE_IOT_HUB.
-# CONFIG_AZURE_IOT_HUB=y
-# CONFIG_AZURE_IOT_HUB_DPS=y
-# CONFIG_AZURE_IOT_HUB_AUTO_DEVICE_TWIN_REQUEST=y
-# CONFIG_AZURE_IOT_HUB_DEVICE_ID_APP=y
-# CONFIG_AZURE_IOT_HUB_MQTT_PAYLOAD_BUFFER_LEN=2048
-# CONFIG_AZURE_IOT_HUB_MQTT_RX_TX_BUFFER_LEN=2048
-# CONFIG_AZURE_IOT_HUB_TOPIC_PROPERTY_BAG_PREFIX=n
-# CONFIG_AZURE_IOT_HUB_STACK_SIZE=4096
-# CONFIG_AZURE_FOTA=y
-# CONFIG_AZURE_FOTA_TLS=y
-
-# ## Azure IoT Hub options that must be configured.
-# CONFIG_AZURE_IOT_HUB_DPS_HOSTNAME="global.azure-devices-provisioning.net"
-# CONFIG_AZURE_IOT_HUB_DPS_ID_SCOPE=""
-# CONFIG_AZURE_IOT_HUB_SEC_TAG=11
-# CONFIG_AZURE_FOTA_SEC_TAG=11
-
-# nRF Cloud
-CONFIG_NRF_CLOUD=y
-CONFIG_NRF_CLOUD_MQTT=y
-CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y
-CONFIG_NRF_CLOUD_SEND_TIMEOUT=y
-
-# GPS
-CONFIG_GPS_MODULE_NMEA=y
```

Comment:
`CONFIG_GPS_MODULE_NMEA` doesn't seem to be enabled anywhere anymore. It is needed with nRF Cloud, I think.

Reply:
`GPS_MODULE_NMEA` is the default if building with `NRF_CLOUD_MQTT`.
See the changes in applications/asset_tracker_v2/src/modules/Kconfig.gps_module